packages/flutter/lib/src/widgets/actions.dart

```diff
@@ -222,7 +226,8 @@ abstract class Action<T extends Intent> with Diagnosticable {
           exception: exception,
           stack: stack,
           library: 'widgets library',
-          context: ErrorDescription('while dispatching notifications for $runtimeType'),
+          context: ErrorDescription(
+              'while dispatching notifications for $runtimeType'),
```

Comment:
I think this formatting change is actually against Flutter's usual style.  Usually it would be:

```dart
context: ErrorDescription(
  'while dispatching notifications for $runtimeType',
),
```